src/transformers/tokenization_utils.py

```diff
@@ -919,10 +921,10 @@ def encode_plus(
         truncation_strategy="longest_first",
         pad_to_max_length=False,
         return_tensors=None,
-        return_token_type_ids=True,
-        return_attention_mask=True,
-        return_overflowing_tokens=False,
-        return_special_tokens_mask=False,
+        return_token_type_ids=None,
```

Comment:
maybe explicitly define them as: `Optional[bool] = None`?

Reply:
Yes, I'll update the typings, doc & tests if @mfuntowicz and @thomwolf agree that this is the best way to do it.

Reply:
Putting `None` means that we can safely identify if the user has passed a value different from the default. If a value is `None`, then we can rely on the `return_outputs` attribute to return this value or not.
If it is not `None`, then its value is absolute (as it's an argument that was passed by the user).

Reply:
Ok I see, then I guess a clear typing like @julien-c suggested would be good